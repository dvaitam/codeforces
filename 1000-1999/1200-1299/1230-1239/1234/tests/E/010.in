9 9
9 8 6 3 4 2 7 4 8
