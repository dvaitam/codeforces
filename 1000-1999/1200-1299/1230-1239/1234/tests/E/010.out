21 23 23 27 17 23 17 32 28
